api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -8081,3 +8082,29 @@ message StableUNIXUser {
   string username = 1;
   int32 uid = 2;
 }
+
+// AWSICResourceSync is emitted when the AWS Identity Center integration
+// imports resources from Identity Center and reconciles them to Teleport.
+message AWSICResourceSync {
+  // Metadata is the event metadata.
+  Metadata metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+  // Total imported AWS accounts.
+  int32 total_account = 2 [(gogoproto.jsontag) = "total_account,omitempty"];
+  // Total account assignments imported from Identity Center. Account assignments
+  // are permission's assigned to users or group.
+  int32 total_account_assignment = 3 [(gogoproto.jsontag) = "total_account_assignment,omitempty"];
+  // Total user groups imported from Identity Center.
+  int32 total_user_group = 4 [(gogoproto.jsontag) = "total_user_group,omitempty"];
+  // Total permission sets imported from Identity Center.
+  int32 total_permission_set = 5 [(gogoproto.jsontag) = "total_permission_set,omitempty"];
+  // Status is a resource sync status.
+  Status Status = 6 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
```

Comment:
Will we have an equivalent of:
```
// OktaSyncFailureEvent is emitted when the Okta synchronization fails.
	OktaSyncFailureEvent = "okta.sync.failure"
```	
where the failure event has a different FE code prefix do display failed even with Warning Icon ?

The Status Status is only validated when the sync fails.

Reply:
Implemented dedicated failure event https://github.com/gravitational/teleport/pull/52059/commits/bd1104570c3086009bc96b717995426fcb342430